appknox/tests/test_client.py

```diff
@@ -324,3 +348,13 @@ def test_write_data_to_file_raises_exception_if_file_is_not_writeable(self):
                 ValueError, msg=("Failed to write data to {}".format(file_path))
             ):
                 self.ap_client.write_data_to_file(b"1,2,3", file_path)
+
+    def test_write_data_to_file_should_work_if_only_filename_is_provided_in_file_path(
+        self,
+    ):
+        file_path = "report_summary.xlsx"
```

Comment:
Before calling `write_data_to_file`, create a temporary directory and change the current directory to that temporary directory.